arch/arm/include/stm32/stm32g47xxx_irq.h

```diff
@@ -0,0 +1,204 @@
+#define STM32_IRQ_DMA2CH4     (STM32_IRQ_FIRST+59)  /* 59: DMA2 channel 4 global interrupt */
+
+#define STM32_IRQ_DMA2CH5     (STM32_IRQ_FIRST+60)  /* 60: DMA2 channel 5 global interrupt */
+#define STM32_IRQ_ADC4        (STM32_IRQ_FIRST+61)  /* 61: ADC4 global interrupt */
+#define STM32_IRQ_ADC5        (STM32_IRQ_FIRST+62)  /* 62: ADC5 global interrupt */
+#define STM32_IRQ_UCPD        (STM32_IRQ_FIRST+63)  /* 63: UCPD global interrupt */
+#define STM32_IRQ_COMP123     (STM32_IRQ_FIRST+64)  /* 64: COMP1, COMP2, and COMP3 interrupts */
+#define STM32_IRQ_COMP456     (STM32_IRQ_FIRST+65)  /* 65: COMP4, COMP5, and COMP6 interrupts */
+#define STM32_IRQ_COMP7       (STM32_IRQ_FIRST+66)  /* 66: COMPP7 interrupt */
+#define STM32_IRQ_HRTIM1MST   (STM32_IRQ_FIRST+67)  /* 67: HRTIM1 master timer interrupt */
+#define STM32_IRQ_HRTIM1TIMA  (STM32_IRQ_FIRST+68)  /* 68: HRTIM1 timer A interrupt */
+#define STM32_IRQ_HRTIM1TIMB  (STM32_IRQ_FIRST+69)  /* 69: HRTIM1 timer B interrupt */
+
+#define STM32_IRQ_HRTIM1TIMC  (STM32_IRQ_FIRST+70)  /* 70: HRTIM1 timer C interrupt */
+#define STM32_IRQ_HRTIM1TIMD  (STM32_IRQ_FIRST+71)  /* 71: HRTIM1 timer D interrupt */
+#define STM32_IRQ_HRTIM1TIME  (STM32_IRQ_FIRST+72)  /* 72: HRTIM1 timer E interrupt */
+#define STM32_IRQ_HRTIM1FLT   (STM32_IRQ_FIRST+73)  /* 73: HRTIM1 fault interrupt */
+#define STM32_IRQ_HRTIM1TIMF  (STM32_IRQ_FIRST+74)  /* 74: HRTIM1 timer E interrupt */
+#define STM32_IRQ_CRS         (STM32_IRQ_FIRST+75)  /* 75: CRS (Clock Recovery System) global interrupt */
+#define STM32_IRQ_SAI1        (STM32_IRQ_FIRST+76)  /* 76: SAI4 global interrupt */
+#define STM32_IRQ_TIM20BRK    (STM32_IRQ_FIRST+77)  /* 77: TIM20 Break, Transition error, Index error interrupt */
+#define STM32_IRQ_TIM20UP     (STM32_IRQ_FIRST+78)  /* 78: TIM20 Update interrupt */
+#define STM32_IRQ_TIM20TRGCOM (STM32_IRQ_FIRST+79)  /* 79: TIM20 Trigger, Commutation, Direction Change, and Index interrupt */
+
+#define STM32_IRQ_TIM20CC     (STM32_IRQ_FIRST+80)  /* 80: TIM20 Capture Compare interrupt */
+#define STM32_IRQ_FPU         (STM32_IRQ_FIRST+81)  /* 81: FPU global interrupt */
+#define STM32_IRQ_I2C4EV      (STM32_IRQ_FIRST+82)  /* 82: I2C4 event interrupt */
+#define STM32_IRQ_I2C4ER      (STM32_IRQ_FIRST+83)  /* 83: I2C4 error interrupt */
+#define STM32_IRQ_SPI4        (STM32_IRQ_FIRST+84)  /* 84: SPI4 global interrupt */
+#define STM32_IRQ_AES         (STM32_IRQ_FIRST+85)  /* 85: AES global interrupt */
+#define STM32_IRQ_FDCAN2_0    (STM32_IRQ_FIRST+86)  /* 86: FDCAN2 interrupt 0 */
+#define STM32_IRQ_FDCAN2_1    (STM32_IRQ_FIRST+87)  /* 87: FDCAN2 interrupt 1 */
+#define STM32_IRQ_FDCAN3_0    (STM32_IRQ_FIRST+88)  /* 88: FDCAN3 interrupt 0 */
+#define STM32_IRQ_FDCAN3_1    (STM32_IRQ_FIRST+89)  /* 89: FDCAN3 interrupt 1 */
+
+#define STM32_IRQ_RNG         (STM32_IRQ_FIRST+90)  /* 90: RNG global interrupt */
+#define STM32_IRQ_LPUART1     (STM32_IRQ_FIRST+91)  /* 91: LPUART1 */
```

Comment:
Good catch! Fixed in d8f29f6263e4d22f0c3eb15547f14bae6f890eb2

Reply:
> @hartmannathan - I can look at it again this weekend.

@davids5 - Thanks! Once you finish reviewing, assuming it looks good, feel free to remove the [DO NOT MERGE] from the title and do the honors!

I've confirmed that it still builds, flash-programs, and runs on the board, following all the cumulative changes made since I opened the PR. It's already rebased on latest master (as of this writing)...

Thanks for all your work so far!